include/seqan3/io/sequence_file/input_options.hpp

```diff
@@ -26,6 +26,8 @@ struct sequence_file_input_options
 {
     //!\brief Read the ID string only up until the first whitespace character.
     bool truncate_ids = false;
+    //!\brief Read the complete header to id.
+    bool complete_header = false;
```

Comment:
please prefix the variable name with `embl_` so it is clear that its a NO-OP for other formats.